### PR TITLE
added npm@8 to sw req'ments

### PR DESCRIPTION
**Pull request by JamesBauman**
Signed-off-by: jamesbauman <james.bauman@broadcom.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Request originated from Fernando. Add npm@8 to the note in SCS plugin. Commit to docs-staging branch.

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/cli-scsplugin.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ For detailed command, actions, and option documentation for this plug-in, see ou
 
 Before you install the plug-in, meet the software requirements in [Software requirements for Zowe CLI plug-ins](cli-swreqplugins.md).
 
-**Note:** If you're using npm@7, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 users: `npm install -g prebuild-install`
+**Note:** If you're using npm@7 or npm@8, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 and npm@8 users: `npm install -g prebuild-install`
```
Comment (IgorCATech) [inline docs/user-guide/cli-scsplugin.md:27]: Not sure, why this is arranged as a note. Maybe you should formulate it as part of the pre-req. 

```suggestion
(npm@7 or npm@8 only) Ensure that you have an internet connection when you install the Secure Credential Store plug-in. If Windows requires npm@7 and npm@8, issue the following command:

```
`npm install -g prebuild-install`
```
```

**Comment by JamesBauman** [comment]
@IgorCATech  - Thanks for the suggestion. Please review.

File: docs/user-guide/cli-scsplugin.md [hunk 2]
```diff
@@ -24,7 +24,14 @@ For detailed command, actions, and option documentation for this plug-in, see ou
 
 Before you install the plug-in, meet the software requirements in [Software requirements for Zowe CLI plug-ins](cli-swreqplugins.md).
 
-**Note:** If you're using npm@7 or npm@8, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 and npm@8 users: `npm install -g prebuild-install`
+**npm@7 and npm@8 Requirements:**
+
+ * Ensure that your computer is connected to the Internet when you install the Secure Credential Store plug-in.
+ * On Windows operating systems, issue the following command before you issue the command to install the Secure Credential Store plug-in:
```
Comment (IgorCATech) [inline docs/user-guide/cli-scsplugin.md:30]: maybe just 

"...issue the following command before you install the Secure Credential Store plug-in:"
?

It sounds a bit repetitive with the word "command" mentioned twice :)

**Review by IgorCATech — COMMENTED** [review]
Thanks for introducing changes, Jim! I've left a minor comment in your commit. Otherwise, looks good 👍

**Comment by JamesBauman** [comment]
OK to merge per Ashley

**Pull request merged**